__tests__/nc_news_db.test.js

```diff
@@ -90,14 +91,65 @@ describe("GET /api/articles/:articles_id", () => {
 	});
 });
 
+describe("GET /api/articles/:articles_id/comments", () => {
+	it("status: 200, should respond with an array of comments from the given article_id", () => {
+		return request(app)
+			.get("/api/articles/1/comments")
+			.expect(200)
+			.then(({ body }) => {
+				const commentsArray = body.comments;
+				commentsArray.forEach((comment) => {
+					expect(comment).toMatchObject({
+						comment_id: expect.any(Number),
+						votes: expect.any(Number),
+						created_at: expect.any(String),
+						author: expect.any(String),
+						body: expect.any(String),
+						article_id: expect.any(Number),
+					});
+				});
+			});
+	});
+
+	it("status; 200, should respond with all articles in descending order by date", () => {
+		return request(app)
+			.get("/api/articles")
+			.expect(200)
+			.then(({ body }) => {
+				const articlesArray = body.articles;
+				expect(articlesArray).toBeSortedBy("created_at", {
+					descending: true,
+				});
+			});
+	});
+
+	it("status: 400, should respond with a 400 message when passed an invalid article_id", () => {
+		return request(app)
+			.get("/api/articles/thisisinvalid/comments")
+			.expect(400)
+			.then(({ body }) => {
+				expect(body.message).toBe("not a valid article ID");
+			});
+	});
+
+	it("status: 404, should respond with a 404 message when passed a valid but none existent article_id", () => {
+		return request(app)
+			.get("/api/articles/808/comments")
+			.expect(404)
+			.then(({ body }) => {
+				expect(body.message).toBe("no article found under article_id 808");
+			});
+	});
+});
```

Comment:
One missing happy path test status 200 for article with no associated comments. Returns empty array